tests/sentry/incidents/test_metric_issue_detector_handler.py

```diff
@@ -0,0 +1,262 @@
+from datetime import UTC, datetime, timedelta
```

Comment:
diff got kinda large when I moved this because I copy/pasted the tests into new files rather than moving the whole original file and then copy/pasting the post process tests, but nothing has changed